ebookmaker/EbookMaker.py

```diff
@@ -273,149 +276,153 @@ def add_local_options (ap):
 
-    ap.add_argument (
+    ap.add_argument(
         "--validate",
         dest     = "validate",
         action   = "count",
         help     = "validate epub through epubcheck")
 
-    ap.add_argument (
+    ap.add_argument(
         "--section",
         metavar  = "TAG.CLASS",
         dest     = "section_tags",
         default  = [],
         action   = "append",
         help     = "split epub on TAG.CLASS")
 
-    ap.add_argument (
+    ap.add_argument(
         "--packager",
         dest    = "packager",
         choices = ['ww', 'gzip'],
         default = None,
         help    = "PG internal use only: which packager to use (default: none)")
 
-    ap.add_argument (
+    ap.add_argument(
         "--cover",
         dest    = "coverpage_url",
         default = None,
         help    = "use the cover specified by an absolute url")
 
-    ap.add_argument (
+    ap.add_argument(
         "--generate_cover",
         dest    = "generate_cover",
         action  = "store_true",
         help    = "if no cover is specified by the source, or as an argument, generate a cover")
 
-    ap.add_argument (
+    ap.add_argument(
         "--jobs",
         dest    = "is_job_queue",
         action  = "store_true",
         help    = "PG internal use only: read pickled job queue from stdin")
 
-    ap.add_argument (
+    ap.add_argument(
         "--extension-package",
         metavar  = "PYTHON_PACKAGE",
         dest    = "extension_packages",
         default = [],
         action  = "append",
         help    = "PG internal use only: load extensions from package")
 
-    ap.add_argument (
+    ap.add_argument(
         "url",
         help    = "url of file to convert")
 
 
-def open_log (path):
+def open_log(path):
     """ Open a logfile in the output directory. """
 
-    handler = logging.FileHandler (path, "a")
-    handler.setFormatter (Logger.CustomFormatter (Logger.LOGFORMAT))
-    handler.setLevel (logging.DEBUG)
-    logging.getLogger ().addHandler (handler)
+    handler = logging.FileHandler(path, "a")
+    handler.setFormatter(Logger.CustomFormatter(Logger.LOGFORMAT))
+    handler.setLevel(logging.DEBUG)
+    logging.getLogger().addHandler(handler)
     return handler
 
 
-def close_log (handler):
+def close_log(handler):
     """ Close logfile handler. """
 
-    logging.getLogger ().removeHandler (handler)
-    handler.close ()
+    logging.getLogger().removeHandler(handler)
+    handler.close()
 
 
-def do_job (job):
+def do_job(job):
     """ Do one job. """
 
     log_handler = None
     Logger.ebook = job.ebook
     if job.logfile:
-        log_handler = open_log (os.path.join (job.outputdir, job.logfile))
+        log_handler = open_log(os.path.join(job.outputdir, job.logfile))
 
-    debug ('=== Building %s ===' % job.type)
+    debug('=== Building %s ===' % job.type)
 
     try:
         if job.url:
-            spider = Spider.Spider ()
+            spider = Spider.Spider()
+            if job.url[:4]=='http':
```

Comment:
any reason not to use job.url.startswith('http') ?

Reply:
I thought startwith() is sudo code.

Reply:
Haha that's why I like Python.